### PR TITLE
Adding scalafix conf

### LINKED ISSUE #75: Define scalafix rules
A good example can be taken from [here](https://github.com/zio/zio).

### PR DESCRIPTION
**Pull request by shankarshastri**
Fixes #75

### CODE REVIEW

**Comment by shankarshastri** [comment]
@mijicd , Please review.

File: build.sbt [hunk 1]
```diff
@@ -14,14 +14,22 @@ inThisBuild(
...
+    semanticdbVersion := scalafixSemanticdb.revision,
+    scalafixDependencies += "com.nequissimus" %% "sort-imports" % "0.5.0"
   )
 )
 
 addCommandAlias("fmt", "all scalafmtSbt scalafmt test:scalafmt")
 addCommandAlias("check", "all scalafmtSbtCheck scalafmtCheck test:scalafmtCheck")
 addCommandAlias("testJVM", ";redis/test;benchmarks/test:compile")
 addCommandAlias("testJVM211", ";redis/test")
+addCommandAlias("fix", "all compile:scalafix test:scalafix")
```
Comment (mijicd) [inline build.sbt:28]: Let's add:

```
addCommandAlias("prepare", "fix; fmt")
```

File: build.sbt [hunk 2]
```diff
@@ -56,5 +64,8 @@ lazy val benchmarks =
         "dev.profunktor"    %% "redis4cats-effects" % "0.10.2",
         "io.chrisdavenport" %% "rediculous"         % "0.0.5",
         "io.laserdisc"      %% "laserdisc-fs2"      % "0.4.0"
+      ),
+      scalacOptions in Compile := Seq(
```
Comment (mijicd) [inline build.sbt:68]: Is this one necessary?

**Review by mijicd — COMMENTED** [review]
Two minor remarks, otherwise looks great!

File: build.sbt (hunk 2 above)
Comment (shankarshastri) [reply build.sbt:68]: Yes, it's necessary.
```shell
[info] Running scalafix on 10 Scala sources
[error] (benchmarks / Compile / scalafix) scalafix.sbt.InvalidArgument: The Scala compiler option "-Ywarn-unused" is required to use RemoveUnused.
[error] To fix this problem, update your build to use at least one Scala compiler
[error] option like -Ywarn-unused, -Xlint:unused (2.12.2 or above), or -Wunused (2.13 only)
[error] Total time: 10 s, completed 20-Aug-2020, 2:00:11 pm
```
As mentioned in the below document, it needs to have that scala compiler option.
https://scalacenter.github.io/scalafix/docs/rules/RemoveUnused.html#installation

File: build.sbt (hunk 1 above)
Comment (shankarshastri) [reply build.sbt:29]: Done. Added !!

File: build.sbt (hunk 2 above)
Comment (mijicd) [reply build.sbt:69]: I see, thanks!

**Review by mijicd — APPROVED** [review]

**Pull request merged**